tortoise/backends/base/schema_generator.py

```diff
@@ -437,7 +437,12 @@ def get_create_schema_sql(self, safe: bool = True) -> str:
                     if t["references"].issubset(created_tables | {t["table"]})
                 )
             except StopIteration:
-                raise ConfigurationError("Can't create schema due to cyclic fk references")
+                t = tables_to_create[0]
```

Comment:
This raised an index error for me. Should probably do a try catch for this also, and what does empty `tables_to_create` mean in this context?